packages/manager/apps/catalog-revamp/src/components/ButtonTooltip/ButtonTooltip.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { OsdsIcon } from '@ovhcloud/ods-components/icon/react/';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react/';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_BUTTON_TYPE,
+} from '@ovhcloud/ods-components/button/';
+import { ODS_ICON_SIZE, ODS_ICON_NAME } from '@ovhcloud/ods-components/icon/';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+
+import './ButtonTooltip.scss';
+
+interface TooltipItem {
+  label: string;
+}
+
+interface ButtonTooltipProps {
+  tooltipContent: TooltipItem[];
+}
+
+const ButtonTooltip: React.FC<ButtonTooltipProps> = (props) => {
+  const { tooltipContent } = props;
```

Comment:
```suggestion
const ButtonTooltip: React.FC<ButtonTooltipProps> = ({ tooltipContent }) => {
```

Reply:
Done, but I did not code it, it was in the generator
We have to fix it in packages/manager/core/generator/app/templates/src/components/ButtonTooltip/ButtonTooltip.tsx